components/06-examples/machtigen.hbs

```diff
@@ -0,0 +1,78 @@
+        <div class="row">
+          <div class="col-md-12">
+            <ul class="nav nav-tabs">
+              <li class="active">
+                <a href="machtigen.html">Voor mezelf</a>
+              </li>
+              <li >
+                <a href="machtigen1.html">Voor een ander</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <div class="row dso-rich-content">
+          <div class="col-sm-12">
+            <div class="dso-highlight-box dso-border" style="margin: 16px 0px 8px 0px"> 
+              <div class="row">
+                <div class="col-md-2 col-sm-2  col-xs-4 text-center"><img src="{{path '/images/Digid.png' }}" alt="Digid Logo" style="width:60px; height:60px;"></div>
+                <div class="col-md-10 col-sm-4 col-xs-4 ">
+                  <button type="button" class="btn btn-link" >
+                    <span>Inloggen met digID</span><svg class="di di-angle-right">
+                    <use href="../../dso-icons.svg#angle-right" />
+                    </svg>
+                  </button>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="col-sm-12">
+            <div class="dso-highlight-box dso-border" style="margin: 8px 0px 8px 0px">
+              <div class="row">
+                <div class="col-md-2 col-sm-2 col-xs-4 text-center" ><img src="{{path '/images/Eherkenninglogo.png' }}" alt="Eherkenning logo" style="width:60px; height:60px;"></div>
+                <div class="col-md-10 col-sm-4 col-xs-4">
+                  <button type="button" class="btn btn-link " >
+                    <span>Inloggen met eHerkenning</span>
+                    <svg class="di di-angle-right">
+                      <use href="../../dso-icons.svg#angle-right" />
+                    </svg>
+                  </button>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="col-sm-12">
+            <div class="dso-highlight-box dso-border" style="margin: 8px 0px 8px 0px">
+              <div class="row">
+                <div class="col-md-2 col-sm-2 col-xs-4 text-center"><img src="{{path '/images/EU.png' }}" alt="Afbeelding EU vlag" style="margin: auto 0px; width:60px; height:60px;"></div>
+                <div class="col-md-10 col-sm-4 col-xs-4">
+                  <button type="button" class="btn btn-link " >
+                    <span>Inloggen met Europees erkend inlogmiddel</span>
+                    <svg class="di di-angle-right">
+                      <use href="../../dso-icons.svg#angle-right" />
+                    </svg>
+                  </button>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="col-sm-12">
+            <p>Heeft u geen gegevens voor bovenstaande inlogmogelijkheden? Probeer het dan met een Omgevingsloket-ID. <a>Inloggen met Omgevingsloket-id</a> of <a>Vraag hier een Omgevingsloket-ID aan</a>. 
```

Comment:
Deze `p` zou dus in een `.dso-rich-content` mogen staan, ik denk dat dit de `.dso-rich-content` is die op de `.row` staat